libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -253,16 +251,14 @@ export const DealTicket = ({
       side: normalizedOrder.side,
     });
   }
-  const { data: positionEstimate } = useEstimatePositionQuery({
-    variables: {
-      marketId: market.id,
-      openVolume,
-      orders,
-      collateralAvailable:
-        marginAccountBalance || generalAccountBalance ? balance : undefined,
-    },
+
+  const positionEstimate = usePositionEstimate({
+    marketId: market.id,
+    openVolume,
+    orders,
+    collateralAvailable:
+      marginAccountBalance || generalAccountBalance ? balance : undefined,
     skip: !normalizedOrder,
-    fetchPolicy: 'no-cache',
```

Comment:
I dont think you need the new hook if you change the fetch policy to 'cache-and-network'. This should result in the same behaviour

Reply:
I can't agree: by the any change of order amounts we send new set of variables what break the cache. We could prepare some extended cache rule for Estimates, but sice they don't have explicity id, have a rich various set of variables it would be quite complicated. I think new hook isn't too high price for robust cache in this important place of application